CreateNotificationActivity/handler.ts

```diff
@@ -169,8 +171,17 @@ export const getCreateNotificationActivityHandler = (
   const isEmailBlockedForService =
     blockedInboxOrChannels.indexOf(BlockedInboxOrChannelEnum.EMAIL) >= 0;
 
-  // then we retrieve the optional email from the user profile
-  const maybeEmailFromProfile = getEmailAddressFromProfile(profile);
+  // If the message is sent to the SANDBOX_FISCAL_CODE we consider it a test message
+  // so we send the email notification to the email associated to the user owner
+  // of the sender service (the one registered in the developer portal).
+  // Otherwise we try to get the email from the user profile.
+  const maybeNotificationEmailAddress: Option<NotificationChannelEmail> =
+    newMessageWithoutContent.fiscalCode === lSandboxFiscalCode
```

Comment:
another way is to assert that this is a SandboxFiscalCode ie.

`SandboxFiscalCode.decode(newMessageWithoutContent.fiscalCode).fold( ... )`

in this way you don't have to pass around the SANDBOX_FISCAL_CODE constant